auction-service/src/test/java/com/artbids/unittest/auction/AddArtUnitTest.java

```diff
@@ -52,6 +51,46 @@ void testAddArtGivenAuctionIdAndArtRequest_whenAddArt_thenAddArtResponse() {
 
         // then
         assertNotNull(response);
+        assertEquals(artRequestDto.getProductName(), response.getProductName());
+        assertEquals(artRequestDto.getProductDescription(), response.getProductDescription());
+        assertEquals(artRequestDto.getStartingPrice(), response.getStartingPrice());
+        assertEquals(artRequestDto.getArtistName(), response.getArtistName());
+        assertEquals(artRequestDto.getArtistDescription(), response.getArtistDescription());
+        assertEquals(artRequestDto.getPriceIncreaseRate(), response.getPriceIncreaseRate());
+        assertArrayEquals(artRequestDto.getImageData(), response.getImageData());
+
+        verify(auctionItemRepository, times(1)).save(any(AuctionItem.class));
+    }
+
+    private AddArtRequestDto createAddArtRequestDto() {
```

Comment:
If you want to use a private/helper method you should receive parameters to be generic. That is not useful.